test/jdk/java/nio/file/Files/InputStreamTest.java

```diff
@@ -22,37 +22,126 @@
  */
 
 /* @test
- * @bug 8227609
+ * @bug 8227609 8233451
  * @summary Test of InputStream and OutputStream created by java.nio.file.Files
- * @library ..
+ * @library .. /test/lib
+ * @build jdk.test.lib.Platform
+ * @run junit InputStreamTest
```

Comment:
I think this will need --enable-native-access=ALL-UNNAMED.

Reply:
It works as is in the CI.